inference_sdk/http/utils/aliases.py

```diff
@@ -54,6 +54,18 @@
     "florence-2-large": "florence-pretrains/2",
 }
 
+OCR_ENDPOINTS = {
+    "doctr": "/doctr/ocr",
```

Comment:
TODO (in separate change): add `/ocr/doctr` and enter `doctr/ocr` into deprecation